### PR TITLE
[15.0][MIG] purchase_request_operating_unit

### PR DESCRIPTION
**Pull request by Saran440**
This PR continue from https://github.com/OCA/operating-unit/pull/493

- Delete domain users - from comment https://github.com/OCA/operating-unit/pull/493#discussion_r1091628834
- Update function - from comment https://github.com/OCA/operating-unit/pull/493#discussion_r1118235227

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Functional LG!

**Comment by AaronHForgeFlow** [comment]
@JuanyDForgeFlow can you review this one?

**Review by JuanyDForgeflow — APPROVED** [review]
LGTM

File: purchase_request_operating_unit/wizard/purchase_request_line_make_purchase_order_view.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--Copyright 2016-19 Eficent Business and IT Consulting Services S.L.-->
```
Comment (AaronHForgeFlow) [inline purchase_request_operating_unit/wizard/purchase_request_line_make_purchase_order_view.xml]: Sorry for the nitpicking:

```suggestion
<!--Copyright 2016-19 ForgeFlow S.L.-->
```

File: purchase_request_operating_unit/wizard/purchase_request_line_make_purchase_order_view.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<!--(http://www.eficent.com)-->
```
Comment (AaronHForgeFlow) [inline purchase_request_operating_unit/wizard/purchase_request_line_make_purchase_order_view.xml]:
```suggestion
<!--(http://www.forgeflow.com)-->
```

File: purchase_request_operating_unit/wizard/purchase_request_line_make_purchase_order.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Copyright 2016-19 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
```
Comment (AaronHForgeFlow) [inline purchase_request_operating_unit/wizard/purchase_request_line_make_purchase_order.py]:
```suggestion
# Copyright 2016-19 ForgeFlow S.L.
```

File: purchase_request_operating_unit/tests/test_purchase_request_to_rfq_operating_unit.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# Copyright 2016-19 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
```
Comment (AaronHForgeFlow) [inline purchase_request_operating_unit/tests/test_purchase_request_to_rfq_operating_unit.py]:
```suggestion
# Copyright 2016-19 ForgeFlow S.L.
```

File: purchase_request_operating_unit/security/purchase_security.xml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--Copyright 2016-19 Eficent Business and IT Consulting Services S.L.-->
+<!--(http://www.eficent.com)-->
```
Comment (AaronHForgeFlow) [inline purchase_request_operating_unit/security/purchase_security.xml]: Same here

File: purchase_request_operating_unit/tests/test_purchase_request_operating_unit.py [hunk 6]
```diff
@@ -0,0 +1,109 @@
+# Copyright 2016-19 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline purchase_request_operating_unit/tests/test_purchase_request_operating_unit.py]: Same here

**Review by AaronHForgeFlow — COMMENTED** [review]
Sorry for the nitpicking, can you change that before the merge?

**Comment by Saran440** [comment]
@AaronHForgeFlow Okay, I fixed all file. Can you check again, please?

**Comment by AaronHForgeFlow** [comment]
@Saran440 thank you!

/ocabot merge nobump

**Pull request merged**